src/components/TreebankService/TreebankService.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import {
+  MessagingService,
+  ResponseMessage,
+  WindowIframeDestination as Destination,
+} from 'alpheios-messaging/dist/dev/alpheios-messaging';
```

Comment:
I believe we can just import alpheios-messaging now, because we fixed the package.json 
https://github.com/alpheios-project/alpheios-messaging/commit/622bf92c9393b7e6c588d016a93256494f5e9846

Reply:
Yep, it works without the `/dist/dev/alpheios-messaging`.